sdk/packages/test-utils/src/constants.ts

```diff
@@ -31,8 +31,6 @@ export const TOKEN_ADDRESS =
   '0x73cc960fb6705e9a6a3d9eaf4de94a828cfa6d2a' as const
 export const INVALID_TOKEN_ADDRESS =
   '0x1234000000000000000000000000000000000000' as const
-export const ZERO_ADDRESS =
-  '0x0000000000000000000000000000000000000000' as const
 
 export const MOCK_L1_CHAIN: Chain = {
```

Comment:
General nit: I don't think we need to do capital snake case here. We can just have all our vars / exports be camel case (similar viem / wagmi)

Reply:
Good idea, I renamed them